.azure-pipelines/pipelines.yml

```diff
@@ -39,7 +39,7 @@ stages:
               artifactName: format
             condition: failed()
 
-      - job: docs
+      - job: extension_db
```

Comment:
Are these changes expected? Seems like a merge problem.

Reply:
eek no

sorry about that - im  going to rebase and force push the correct commits

Reply:
should be cleaned up now - apologies, again